java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/jdbc/PostgresSchemaTableName.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.risingwave.connector.jdbc;
+
+public class PostgresSchemaTableName extends SchemaTableName {
+    public PostgresSchemaTableName(String schemaName, String tableName) {
```

Comment:
We already have a `createSchemaTable` in `JdbcDialect`. We can set the default schema name there. This class is unnecessary.